README.adoc

```diff
@@ -786,6 +786,19 @@ puts (x + y)
 puts(x + y)
 ----
 
+=== Space in Brackets Access [[brackets-no-spaces]]
```

Comment:
There's no need for the explicit anchor,  as one will be generated anyways (and it seems to me it'd be more descriptive). All the short unclear anchors are legacy from a time we didn't think enough about them. :-)

Reply:
I get it! That makes sense!